src/components/material.js

```diff
@@ -53,10 +53,16 @@ module.exports.Component = registerComponent('material', {
   },
 
   updateSchema: function (data) {
-    var newShader = data.shader;
-    var currentShader = this.data && this.data.shader;
-    var shader = newShader || currentShader;
-    var schema = shaders[shader] && shaders[shader].schema;
+    var currentShader;
+    var newShader;
```

Comment:
unrelated to PR?

Reply:
fixes test, checks for data / oldData existence